src/app/components/game/game.component.ts

```diff
@@ -68,6 +68,14 @@ export class GameComponent implements OnInit, OnChanges {
         color: string;
         gA: number;
     }[];
+    private drawnArrows: {
```

Comment:
Going to issue a challenge to ya - make this object type EITHER (up to you)

A class (meaning, give it it's own file in `/library`

or an Interface (name it as an interface in the `interface.library.ts` file.

And then reference the object in here with `private drawnArrows: Arrow[];`

Reply:
Boring - this is not a challenge, it's just unnecessary legwork